src/electronic-ids/TLV.hpp

```diff
@@ -99,7 +99,8 @@ struct TLV
     PCSC_CPP_CONSTEXPR_VECTOR TLV operator[](uint32_t find) const
     {
         TLV tlv = child();
-        for (; tlv && tlv.tag != find; ++tlv);
+        for (; tlv && tlv.tag != find; ++tlv)
+            ;
```

Comment:
Our code convention requires always using braces:
```suggestion
        for (; tlv && tlv.tag != find; ++tlv) {}
```